Sources/SourceKitLSP/SourceKitLSPServer.swift

```diff
@@ -553,7 +553,7 @@ package actor SourceKitLSPServer {
     }
 
     let toolchain = await workspace.buildServerManager.toolchain(
-      for: uri,
+      for: uri.buildSettingsFile,
```

Comment:
Is this change necessary? I just checked and it looks like `toolchain(for:in:language:)` doesn’t even use `uri` right now.